app/components/machine/driver-digitalocean/template.hbs

```diff
@@ -1,7 +1,5 @@
 <section class="horizontal-form">
-  <div class="container-fluid">
-    {{partial "host/add-common"}}
-
+  {{#if step1}}
```

Comment:
Wrap the inside of this `{{#if}}` with a `<form>` tag to appease the 1Password gods.